src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -24,15 +24,15 @@
  */
 class LoggerDataCollector extends DataCollector implements LateDataCollectorInterface
 {
-    private $logger;
-    private $containerPathPrefix;
-    private $currentRequest;
-    private $requestStack;
-    private $processedLogs;
+    private ?DebugLoggerInterface $logger = null;
```

Comment:
couldn't we make this unitialized instead of null like in other places

Reply:
We can. 👍🏻